site/content/en/docs/manual/advanced/ai-tools.md

```diff
@@ -44,7 +45,33 @@ In most cases specifying positive points alone is enough to build a polygon.
 - When the object is finished, you can edit it like a polygon.
   You can read about editing polygons in the [Annotation with polygons](/docs/manual/advanced/annotation-with-polygons/) section.
 
-### Detectors
+### Deep extreme cut (DEXTR)
+
+This is an optimized version of the original model, introduced at the end of 2017.
+It uses the information about extreme points of an object to get its mask. The mask then converted to a polygon.
+For now this is the fastest interactor on CPU.
+
+![](/images/dextr_example.gif)
+
+### Feature backpropagating refinement scheme (f-BRS)
+
+The model allows to get a mask for an object using positive points
+(should be clicked on the foreground), and negative points
+(should be clicked on the background, if necessary).
+It is recommended to run the model on GPU, if possible.
+
+![](/images/fbrs_example.gif)
```

Comment:
I think you can cut out the footage of waiting in this gif

Reply:
Done